contract/handler.go

```diff
@@ -0,0 +1,50 @@
+package contract
+
+import (
+	"github.com/seeleteam/go-seele/crypto"
+	"github.com/seeleteam/go-seele/log"
+)
+
+// HandleTransaction handles smart contract transation.
+func HandleTransaction(tx *Transaction) {
+	hash := crypto.Keccak256Hash(tx.payload)
+	if !tx.sig.Verify(hash) {
+		log.Error("Transaction signature is invalid, and will not be archived into blockchain.")
+		return
+	}
+
+	switch tx.txType {
+	case txTypeContractReg:
+		onRegisterContract(tx)
+	case txTypeContractCall:
+		onInvokeContract(tx)
+	case txTypeContractDel:
```

Comment:
add default case

Reply:
unnecessary, do not handle not supported tx type.